File: Trabajos corte 1/Retos sesion del 24 de agosto/retoContador.cc
# include <iostream>
using namespace std;

class objetos
{    
    private:

    static int contador;
  
    public:

    objetos(){contador+=1;}

    int obtenerContador ()
    {
        return contador;
    }
};

int objetos::contador=0;

int main ()
{   

    objetos objeto1;
    objetos objeto2;

    cout<<"el total de objetos es "<<objeto1.obtenerContador()<<endl;
}

File: Trabajos corte 1/Retos sesion del 24 de agosto/retoContador.py
class objeto:
    contador=0
    def __init__(self): #constructor de la clase que se efectua de forma automatica cada que se crea un nuevo objeto
        objeto.contador+=1
    def obtenerContador(self):
        return objeto.contador





objeto1=objeto()
objeto2=objeto()
objeto3=objeto()
print (objeto1.obtenerContador())
print (objeto2.obtenerContador() )